translations/messages+intl-icu.de.yml

```diff
@@ -1014,7 +1028,9 @@ error.copy.mapfile: "Die Einzeichnung der Stellungnahme {externId} konnte nicht
 error.changes.not.saved: "Beim Speichern der Änderungen ist ein Fehler aufgetreten."
 error.date.authored: "Bitte geben Sie ein Verfassungsdatum ein."
 error.in.fields: "Bitte korrigieren Sie Ihre Eingabe in folgenden Feldern: "
+error.phone_or_email: "Bitte geben Sie entweder eine gültige E-Mail-Adresse oder Telefonnummer ein."
 error.startdate.required: "Bitte geben Sie ein Startdatum ein"
+error.title: "Bitte geben Sie Namen für den Kontakt ein."
```

Comment:
the key is too generic for the Text. And I suggest to move it somewhere here:
translations/validators.de.yml
and to be consistant in the wording it then should be "Das Feld 'Name' darf nicht leer sein." I guess

Reply:
solved like so 31d8a46d979e3580af5e74dfb9da8302ed2f11b0